pom.xml

```diff
@@ -513,6 +513,34 @@
         <artifactId>snakeyaml</artifactId>
         <version>1.31</version>
       </dependency>
+
+      <!-- Argo Workflows -->
+      <dependency>
+        <groupId>io.argoproj.workflow</groupId>
+        <artifactId>argo-client-java</artifactId>
+        <version>v3.4.2</version>
+      </dependency>
+      <dependency>
+        <groupId>io.gsonfire</groupId>
+        <artifactId>gson-fire</artifactId>
+        <version>1.8.5</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.okhttp3</groupId>
+        <artifactId>okhttp</artifactId>
+        <version>4.10.0</version>
+      </dependency>
+      <dependency>
+        <groupId>io.kubernetes</groupId>
+        <artifactId>client-java</artifactId>
+        <version>15.0.1</version>
+      </dependency>
```

Comment:
are we repeating them in the POM of `openmetadata-service`? is there a reason to have them here too?

Reply:
that it's me not grasping pom. For my understanding, I have the deps in the parent pom `dependencyManagement` so that I can reference them later on in other poms without specifying the version.

Killed the versions from `openmetadata-service` now. Let me know if there's a better way to handle it, happy to improve it